src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -179,22 +182,27 @@ export default class TaskRoleContainerList extends React.Component {
             },
         ),
       )
-      .catch(err =>
+      .catch(err => {
         this.setState(
           prevState =>
-            prevState.logUrl === logUrl && {
+            prevState.tailLogUrls[logType] === tailLogUrls[logType] && {
               monacoProps: { value: err.message },
             },
-        ),
-      );
+        );
+        this.showContainerTailLog(
```

Comment:
What if log not exists, will it pop up error all the time? This may cause page always reload. Please just check 403 error?